modules/reference/pages/users/user-details.adoc

```diff
@@ -48,6 +48,10 @@ To delete a user:
 
 For instructions to deactivate your own account, see xref:reference:home/user-account-account-deactivation.adoc[].
 
+[NOTE]
+====
+If you delete a user account, any schedules that user created will also be deleted.
```

Comment:
```suggestion
If you delete a user account, any recurring actions that user created will also be deleted.
```

I don't think we need to reference the recurring actions section here, because we're not talking about setting them up, or changing them. Which is also why I moved it into this section, and took it out of the recurring actions section.